generated/python/gapic-google-cloud-logging-v2/google/cloud/gapic/logging/v2/config_service_v2_client.py

```diff
@@ -333,39 +372,56 @@ def update_sink(self,
+        identical to
+        `sinks.create <https://cloud.google.com/logging/docs/api/reference/rest/v2/projects.sinks/create>`_.
+        If the named sink does exist, then this method replaces the following
+        fields in the existing sink with values from the new sink: ``destination``,
+        ``filter``, ``output_version_format``, ``start_time``, and ``end_time``.
+        The updated filter might also have a new ``writer_identity``; see the
+        ``unique_writer_identity`` field.
 
         Example:
           >>> from google.cloud.gapic.logging.v2 import config_service_v2_client
-          >>> from google.cloud.grpc.logging.v2 import logging_config_pb2
+          >>> from google.cloud.proto.logging.v2 import logging_config_pb2
           >>> api = config_service_v2_client.ConfigServiceV2Client()
           >>> sink_name = api.sink_path('[PROJECT]', '[SINK]')
           >>> sink = logging_config_pb2.LogSink()
           >>> response = api.update_sink(sink_name, sink)
 
         Args:
-          sink_name (string): Required. The resource name of the sink to update, including the parent
-            resource and the sink identifier:
+          sink_name (string): Required. The full resource name of the sink to update, including the
+            parent resource and the sink identifier:
 
             ::
 
                 \"projects/[PROJECT_ID]/sinks/[SINK_ID]\"
                 \"organizations/[ORGANIZATION_ID]/sinks/[SINK_ID]\"
 
             Example: ``\"projects/my-project-id/sinks/my-sink-id\"``.
-          sink (:class:`google.cloud.grpc.logging.v2.logging_config_pb2.LogSink`): Required. The updated sink, whose name is the same identifier that appears
-            as part of ``sinkName``.  If ``sinkName`` does not exist, then
+          sink (:class:`google.cloud.proto.logging.v2.logging_config_pb2.LogSink`): Required. The updated sink, whose name is the same identifier that appears
+            as part of ``sink_name``.  If ``sink_name`` does not exist, then
             this method creates a new sink.
-          unique_writer_identity (bool): Optional. Whether the sink will have a dedicated service account returned
-            in the sink's writer_identity. Set this field to be true to export
-            logs from one project to a different project. This field is ignored for
-            non-project sinks (e.g. organization sinks) because those sinks are
-            required to have dedicated service accounts.
+          unique_writer_identity (bool): Optional. See
+            `sinks.create <https://cloud.google.com/logging/docs/api/reference/rest/v2/projects.sinks/create>`_
+            for a description of this field.  When updating a sink, the effect of this
+            field on the value of ``writer_identity`` in the updated sink depends on both
+            the old and new values of this field:
+
+            +   If the old and new values of this field are both false or both true,
```

Comment:
I don't think this is being interpreted correctly by the comment fixer (https://github.com/googleapis/toolkit/blob/master/src/main/java/com/google/api/codegen/py/PythonSphinxCommentFixer.java). But let's not block on this until we need to generate docs.

Tracking as https://github.com/googleapis/toolkit/issues/1033.